documentation/src/main/asciidoc/stories/assembly_setting_up_clusters.adoc

```diff
@@ -16,6 +16,7 @@ include::{topics}/ref_discovery_mping.adoc[leveloffset=+2]
 include::{topics}/ref_discovery_gossip_router.adoc[leveloffset=+2]
 include::{topics}/ref_discovery_jdbc.adoc[leveloffset=+2]
 include::{topics}/ref_discovery_dns_ping.adoc[leveloffset=+2]
+include::{topics}/proc_os_configuring_clusters.adoc[leveloffset=+2]
```

Comment:
I think any additional content, e.g. example services, should be included as part of `ref_discovery_dns_ping.adoc`.